package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "nhs-notify-web-template-management",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "test:unit": "jest",
+    "cypress:open": "cypress open",
+    "test:e2e": "cypress run",
+    "test:accessibility": "pa11y-ci -c ./tests/accessibility/.pa11y-ci.js",
+    "pretest:e2e:local": "npm run app:start",
+    "test:e2e:local": "npm run app:wait && npm run test:e2e",
+    "posttest:e2e:local": "npm run app:stop",
+    "pretest:accessibility:local": "npm run app:start",
+    "test:accessibility:local": "npm run app:wait && npm run test:accessibility",
+    "post:accessibility:local": "npm run app:stop",
+    "app:start": "pm2 start npm -- start",
+    "app:wait": "wait-on -l http://localhost:3000",
+    "app:stop": "pm2 kill"
+  },
+  "dependencies": {
+    "@aws-amplify/adapter-nextjs": "^1.2.4",
+    "@aws-amplify/ui-react": "^6.1.12",
+    "aws-amplify": "^6.3.6",
+    "jose": "^5.4.0",
```

Comment:
We don't need jose yet (that was for producing the POC auth token).

Reply:
Will take them off